<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LdapAuthenticator</class>
 <widget class="QMainWindow" name="LdapAuthenticator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1466</width>
    <height>906</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LDAP Authenticator</string>
  </property>
  <property name="styleSheet">
   <string>
/* Main window background: Dark Gray */
QMainWindow {
    background: #555555;              /* deep dark gray background */
    font-family: &quot;Segoe UI&quot;, Roboto, Arial, sans-serif;
    font-size: 14px;
    color: #E0E0E0;                   /* very light gray text */
}

/* Input Fields */
QLineEdit {
    background: #777777;              /* darker input background */
    border: 2px solid #444444;
    border-radius: 6px;
    padding: 6px;
    color: #E0E0E0;
}
QLineEdit:focus {
    border-color: #5DADE2;
}

/* Buttons */
QPushButton {
    background-color: #3B82F6;        /* strong blue button */
    color: #FFFFFF;
    border-radius: 6px;
    padding: 8px 16px;
}
QPushButton:hover {
    background-color: #60A5FA;        /* lighter on hover */
}
QPushButton:pressed {
    background-color: #2563EB;        /* deeper blue when pressed */
}

/* Tree Widget */
QTreeWidget {
    background-color: #252525;        /* slightly lighter than main window */
    border: 1px solid #444444;
    border-radius: 4px;
    color: #E0E0E0;
}
QHeaderView::section {
    background-color: #3B82F6;
    color: #FFFFFF;
    padding: 4px;
    font-weight: bold;
}
QTreeWidget::item:nth-of-type(odd) {
    background: #2F2F2F;
}
QTreeWidget::item:selected {
    background-color: #60A5FA;
    color: #FFFFFF;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widgetLogin" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEditHost">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelPassword">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="lineEditCA">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelCA">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CA:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="pushButtonEye">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelHost">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Host:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelUPN">
         <property name="font">
          <font>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Principal</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEditUPN">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButtonCA">
         <property name="text">
          <string>Choose</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>pushButtonEye</zorder>
      <zorder>labelHost</zorder>
      <zorder>lineEditHost</zorder>
      <zorder>lineEditUPN</zorder>
      <zorder>labelPassword</zorder>
      <zorder>lineEditPassword</zorder>
      <zorder>pushButtonLogin</zorder>
      <zorder>labelCA</zorder>
      <zorder>lineEditCA</zorder>
      <zorder>labelUPN</zorder>
      <zorder>pushButtonCA</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widgetHome" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="treeWidgetAttributes">
         <column>
          <property name="text">
           <string>Attribute</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonChangePassword">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonLogout">
         <property name="text">
          <string>Log out</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
